src/vivarium/framework/values/manager.py

```diff
@@ -267,10 +278,13 @@ def get_attribute(self, name: str) -> AttributePipeline:
         """
         if name in self._value_pipelines:
             raise DynamicValueError(f"'{name}' is already registered as a value pipeline.")
-        pipeline = self._attribute_pipelines.get(name) or AttributePipeline(name)
+        pipeline = self._attribute_pipelines.get(name, AttributePipeline(name))
         self._attribute_pipelines[name] = pipeline
         return pipeline
 
+    def get_attribute_pipelines(self) -> dict[str, AttributePipeline]:
+        return self._attribute_pipelines
```

Comment:
It's redundant to have `_attribute_pipelines` accessible via a getter method and a property. We should choose only one of them.